docs/dotnet/dotnet-dns-metrics.md

```diff
@@ -0,0 +1,51 @@
+# Semantic Conventions for DNS metrics emitted by .NET
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for DNS metrics emitted by .NET.
+
+**Disclaimer:** These are initial .NET metric instruments available in .NET 8 but more may be added in the future.
+
+<!-- toc -->
+
+- [DNS metrics](#dns-metrics)
+  * [Metric: `dns.lookups.duration`](#metric-dnslookupsduration)
+
+<!-- tocstop -->
+
+## DNS metrics
+
+### Metric: `dns.lookups.duration`
+
+**Status**: [Experimental][DocumentStatus]
+
+This metric is required.
+
+This metric SHOULD be specified with
+[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/metrics/api.md#instrument-advice)
+of **`[TODO]`**.
```

Comment:
.NET doesn't support advice right now (advice itself is an experimental feature). I don't think we should specify something we aren't implementing.

Reply:
removed